config/locales/de.yml

```diff
@@ -45,14 +45,40 @@ de:
     oauth_text: "Dieses Konto ist durch %{provider} authentifiziert."
     own_projects: "Von dir gestartete Projekte"
     participating_projects: "Projekte, an denen du teilnimmst"
+    projects: "Deine Projekte"
+    no_projects: "Es gibt keine Projekte, an denen du bereits teilnimmst, oder die du veranstaltest."
+    all_projects_pre: "Alle vorhandenen Projekte"
+    show: "anzeigen"
+    all_projects_post: ""
+    new_projects_pre: "Ein neues Projekt"
+    create: "anlegen"
+    new_projects_post: ""
+
+%h3= t('profile.projects')
+
+          .panel-body
+            %h4= t('profile.no_projects')
+
+            %p
+              #{t('profile.all_projects_pre')}
+              %a(href='/')=t('profile.show')
+              #{t('profile.all_projects_post')}
+
+            %p
+              #{t('profile.new-project_pre')}
+              %a(href='/projects/new')=t('profile.create')
+              #{t('profile.new-project_post')}
+
+
```

Comment:
This shouldn't be included in this patch. I know you are removing it again in another patch, but it really shouldn't be there in the first place.